src/mpyl/steps/test/dockertest.py

```diff
@@ -21,8 +22,8 @@ def __init__(self, logger: Logger) -> None:
             logger=logger, meta=meta,
             produced_artifact=ArtifactType.JUNIT_TESTS,
             required_artifact=ArtifactType.NONE,
-            before=IntegrationTestBefore(logger),
-            after=IntegrationTestAfter(logger)
+            before=[IntegrationTestBefore(logger)],
+            after=[ScanDocker(logger), IntegrationTestAfter(logger)]
```

Comment:
Can we do any assertion that this after step succeeded? Or it triggers automatically for the sample projects and that's enough?